coral-service/src/main/java/com/linkedin/coral/coralservice/controller/TranslationController.java

```diff
@@ -30,14 +31,17 @@
 @Service
 @Profile({ "remoteMetastore", "default" })
 public class TranslationController implements ApplicationListener<ContextRefreshedEvent> {
+  @Value("${hiveProps:}")
+  private String hiveProps;
```

Comment:
Let's modify the name to something like `hivePropsLocation`?

Reply:
Ack. Done.